src/test/csrc/difftest/refproxy.h

```
@@ -268,6 +269,14 @@ class RefProxy : public AbstractRefProxy {
     sync_config();
   }
 
+  inline void mem_init(uint64_t dest, void *src, size_t n, bool direction) {
+    if (ref_memcpy_init) {
+      ref_memcpy_init(dest, src, n, direction);
+    } else {
+      ref_memcpy(dest, src, n, direction);
```

Should report an error here `#if NUM_CORES > 1`?

After cbo is implemented, ref_memcpy_init must be implemented either in single-core or multi-core.